app/models/service_template_ansible_playbook.rb

```diff
@@ -151,4 +151,17 @@ def update_catalog_item(options, auth_user = nil)
     end
     super
   end
+
+  def destroy(auth_user = 'system')
```

Comment:
@syncrou I would suggest that we determine the `auth_user` property this way instead of expecting the user to tell us:
```ruby
def destroy
  auth_user = User.current_user || 'system'
```

This keeps the method signature unchanged and gets the one thing we all want; using `super` instead of `super()`.  😉